src/components/Signup/Signup.js

```diff
@@ -33,6 +35,12 @@ const minLength = {
   errorMessageKey: 'validation_messages.error_min_length_2',
 };
 
+/*Extract the page parameter from url*/
+function extractPage(location) {
+  const parsedQuery = location && location.search && queryString.parse(location.search);
+  return (parsedQuery && (parsedQuery['page'] || parsedQuery['page'])) || null;
```

Comment:
```suggestion
  return (parsedQuery && parsedQuery['page']) || null;
```

Reply:
sorry its 'Page' in the second, because in the url can put 'Page' like in origin 'Origin' right?

Reply:
I did not understand.

Reply:
?page='features' or ?Page='features

Reply:
ahh, mayusculas y minusculas? esperá minusculas y listo, no? (creo que las comillas no van)

Reply:
nono era por ejemplo las comillas. Puse la mayuscula pq en origin lo pusieron

Reply:
Creo que la razón es que Origin es un campo bastante viejo, que probablemente esté escrito de distantas maneras en distintos lugares.

Si podés confirmá con @mfantini si es necesario aceptar variaciones de `page` (yo creo que no)

Reply:
Dale joya mañana ya cuando este confirmo! gracias!